### ISSUE TITLE
training output meaning and time

### ISSUE THREAD

**Issue by thefirebanks** [kermitt2/grobid#336]
Hello!

First of all, we are really thankful to all the people involved in this project - it has been the only most reliable tool we could have found for our task: Extracting headers/subheaders and body from documents.

Now, to the concern: I have run createTraining and manually labeled about 30 documents to add to the training set. When I try to retrain the model using "./gradlew train_fulltext", the process seems to be taking a long time (> 3 or 5 hours), and I was wondering if you could please explain what the information shown in the console mean (obj, act, err, time) and if it is supposed to last as many hours as it is taking. I am running the model on a macbook pro of late 2012.

Thank you! 
<img width="1276" alt="screen shot 2018-08-09 at 2 15 22 pm" src="https://user-images.githubusercontent.com/31460539/43918081-141f0390-9be0-11e8-9b45-b7a3a5e45c41.png">

**Comment by thefirebanks** [comment]
Found all the responses we were looking for, thanks!

**Comment by thefirebanks** [comment]
Thank you @kermitt2 ! This worked perfect. Kudos for your tool!

**Comment by epugh** [comment]
If I stop the model training before it is done via a `ctrl -c`, can I still use it?   Is there any way to change the `nbMaxIterations` without changing the java code and recompiling?   I found a setParams method on AbstractTrainer.....

**Comment by epugh** [comment]
Okay, I answered my question on the `ctrl -c`, after doing that, I get the output: `^C* Save the model` and it saves it.   

in `grobid-home/models/header/` I have an updated model.wapiti.   Interestingly, I don't have the previous model, which I thought would be still stored as model.wapiti.old.

The docs on https://grobid.readthedocs.io/en/latest/Training-the-models-of-Grobid/#train-and-evaluate may be out of date, as they suggest that when I generaated the header it will e called "header.wapiti", but instead I have "/models/header/model.wapiti".
